task/pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>test_task.task</groupId>
   <artifactId>task</artifactId>
   <packaging>jar</packaging>
-  <version>1.0</version>
```

Comment:
If the previous version for this was 1.0, should the new version be 2.0 instead of 2.0.0?

Reply:
It's the same